templates/sheets/actor/monster-sheet.html

```diff
@@ -133,105 +151,128 @@ <h1><input name="name" type="text" value="{{actor.name}}" placeholder="Name" /><
-            {{#if (gt system.deathSaves 2)}}<i class="death fas fa-skull-crossbones"></i>.{{/if}}{{#unless (gt system.deathSaves 2)}}<i class="fas fa-skull"></i>.{{/unless}}
-            {{#if (gt system.deathSaves 3)}}<i class="death fas fa-skull-crossbones"></i>.{{/if}}{{#unless (gt system.deathSaves 3)}}<i class="fas fa-skull"></i>.{{/unless}}
-            {{#if (gt system.deathSaves 4)}}<i class="death fas fa-skull-crossbones"></i>.{{/if}}{{#unless (gt system.deathSaves 4)}}<i class="fas fa-skull"></i>.{{/unless}}
-            {{#if (gt system.deathSaves 5)}}<i class="death fas fa-skull-crossbones"></i>.{{/if}}{{#unless (gt system.deathSaves 5)}}<i class="fas fa-skull"></i>.{{/unless}}
-            {{#if (gt system.deathSaves 6)}}<i class="death fas fa-skull-crossbones"></i>.{{/if}}{{#unless (gt system.deathSaves 6)}}<i class="fas fa-skull"></i>.{{/unless}}
-            {{#if (gt system.deathSaves 7)}}<i class="death fas fa-skull-crossbones"></i>.{{/if}}{{#unless (gt system.deathSaves 7)}}<i class="fas fa-skull"></i>.{{/unless}}
-            {{#if (gt system.deathSaves 8)}}<i class="death fas fa-skull-crossbones"></i>.{{/if}}{{#unless (gt system.deathSaves 8)}}<i class="fas fa-skull"></i>.{{/unless}}
-            <a class="death-plus"><i class="fas fa-plus"></i></a>
-        </div>
-        <nav class="sheet-tabs tabs" data-group="primary">
-            <a class="item" data-tab="skills"> {{localize "WITCHER.Monster.SkillTab"}}</a>
-            <a class="item" data-tab="inventory">{{localize "WITCHER.Monster.InventryTab"}} </a>
-            <a class="item" data-tab="details"> {{localize "WITCHER.Monster.DetailsTab"}}</a>
-            <a class="item" data-tab="spells"> {{localize "WITCHER.Monster.SpellsTab"}}</a>
-        </nav>
+            <div class="monster-right-sidebar">
+                <div class="monster-right-top">
+                    <div class="monster-button-roll">
+                        <a class="init-roll">{{localize "WITCHER.Actor.Initiative"}}</a>
+                        <a class="death-roll">{{localize "WITCHER.Actor.SavingThrow"}}</a>
+                    </div>
+                    <div class="monster-button-roll">
+                        <a class="crit-roll">{{localize "WITCHER.Actor.Crit/Fumble"}}</a>
+                        <a class="defence-roll">{{localize "WITCHER.Actor.Defense"}}</a>
+                        {{#if useVerbalCombat}}
+                            <a class="button-roll verbal-button">{{localize "WITCHER.verbalCombat.Title"}}</a>
+                        {{/if}}
+                    </div>
+                </div>
+                <div class="death-section">
+                    <a class="death-minus"><i class="fas fa-minus"></i></a>
+                    {{#if (gt system.deathSaves 0)}}<i class="death fas fa-skull-crossbones"></i>.{{/if}}{{#unless
```

Comment:
same as in the character sheet we could use if else

Reply:
will do